app/component/map/tile-layer/Roadworks.js

```diff
@@ -0,0 +1,84 @@
+import { VectorTile } from '@mapbox/vector-tile';
+import Protobuf from 'pbf';
+import pick from 'lodash/pick';
+
+import { isBrowser } from '../../../util/browser';
+import { drawIcon } from '../../../util/mapIconUtils';
+import glfun from '../../../util/glfun';
+
+const getScale = glfun({
+  base: 1,
+  stops: [[13, 0.8], [20, 1.6]],
+});
+
+class Roadworks {
+  constructor(tile, config) {
+    this.tile = tile;
+    this.config = config;
+
+    this.scaleratio = (isBrowser && window.devicePixelRatio) || 1;
+    this.iconSize = 36 * this.scaleratio * getScale(this.tile.coords.z);
+
+    this.promise = this.fetchWithAction(this.fetchAndDrawStatus);
+  }
+
+  fetchWithAction = actionFn =>
+    fetch(
+      `${this.config.URL.ROADWORKS_MAP}` +
+        `${this.tile.coords.z + (this.tile.props.zoomOffset || 0)}/` +
+        `${this.tile.coords.x}/${this.tile.coords.y}.pbf`,
+    ).then(res => {
+      if (res.status !== 200) {
+        return undefined;
+      }
+
+      return res.arrayBuffer().then(
+        buf => {
+          const vt = new VectorTile(new Protobuf(buf));
+
+          this.features = [];
+
+          // it's called "roadwork" in the tile source but i want to change it to "roadworks" since that is more
+          // consistent
```

Comment:
So this is only for backward compatibility? Could you do all changes so we don't need this any longer?

Reply:
I forgot to push! https://github.com/mfdz/tilelive-roadworks-bw/commit/1ca481244aa5436a036e50cdb5b4fdbcb91ef90a